app/src/main/java/science/apolline/service/sensor/BluetoothLeService.kt

```diff
@@ -0,0 +1,620 @@
+                        } catch (e: NumberFormatException) {
+                            appaData.AltiGPS = 0.0
+                        }
+
+                        //Vitesse(km/h)
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.SpeedGPS = java.lang.Double.parseDouble(buff.substring(debut + 1, fin))
+                        } catch (e: NumberFormatException) {
+                            appaData.SpeedGPS = 0.0
+                        }
+
+                        //nb satellites
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.SatGPS = Integer.parseInt(buff.substring(debut + 1, fin))
+                        } catch (e: NumberFormatException) {
+                            appaData.SatGPS = 0
+                        }
+
+                        //T° Dps310
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        //Pression
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.pression = java.lang.Double.parseDouble(buff.substring(debut + 1, fin)) / 100
+                        } catch (e: NumberFormatException) {
+                            appaData.pression = 0.0
+                        }
+
+                        //T° HDC1080
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.tempe = java.lang.Double.parseDouble(buff.substring(debut + 1, fin))
+                        } catch (e: NumberFormatException) {
+                            appaData.tempe = 0.0
+                        }
+
+                        // Humidité
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.humi = java.lang.Double.parseDouble(buff.substring(debut + 1, fin))
+                        } catch (e: NumberFormatException) {
+                            appaData.humi = 0.0
+                        }
+
+                        // batterie level
+                        debut = fin
+                        fin = buff.indexOf(";", debut + 1)
+                        try {
+                            appaData.bat_volt = java.lang.Double.parseDouble(buff.substring(debut + 1, fin))
+                        } catch (e: NumberFormatException) {
+                            appaData.bat_volt = 0.0
+                        }
+
+                    }
+                    // generate Dates
+                    val calendar = Calendar.getInstance()
+                    val d1 = calendar.time
+                    // écrire dans un fichier
```

Comment:
Be consistent with comments and avoid them whenever useless

Reply:
OK, I'll try to clean the comments off my code  as much as possible.